### ISSUE TITLE
Support for Nano server.

### ISSUE THREAD

**Issue by AndreasSobczyk** [PowerShell/xCertificate#45]
xCertificateImport currently fails on Test-Thumbprint because [System.AppDomain] does not exist in Nano server.
Since GPOs is not supported on Nano server the xCertificateImport would be the perfect mechanism to import Enterprise Root certs into the nano server

**Comment by PlagueHO** [comment]
Good catch @AndreasSobczyk and thanks for raising this. I think we're also going to need to make other changes for this to support Nano Server. ```Import-Certificate``` is used but isn't supported on Nano I think.

So we need to actually have an alternate path through the resource that will call ```certoc.exe```. See [this page](https://technet.microsoft.com/en-us/windows-server-docs/get-started/iis-on-nano-server) for more info.

I'm planning on putting some time into this resource over the weekend so I'll try and get some PR's submitted for this one as well unless someone else gets to it first! :grin:

**Comment by AndreasSobczyk** [comment]
Import-Certificate is actually supported and work now. This is what i have used to add the certificates without DSC :)

**Comment by PlagueHO** [comment]
@AndreasSobczyk - oh cool! That is some great info (and news). I'm working on some other xCertificate issues this weekend so I'll attempt to put time into resolving this.

**Comment by markusknappfr** [comment]
Any news on this issue?

**Comment by shubham1172** [comment]
@AndreasSobczyk `Import-Certificate` is not available on nano server (`mcr.microsoft.com/powershell:lts-7.2-nanoserver-1809`).
